cmake/FindGLFW.cmake

```diff
@@ -0,0 +1,52 @@
+# Finds the OpenGL Graphics Library Framework library
+
+# This file can be configured with the following CMake variables:
+# - GLFW_ROOT_DIR
+# - GLFW_INCLUDE_DIR
+# - GLFW_LIBRARY
+
+# Or, alternatively, the following environment variables:
+# - GLFW_INSTALL_DIR
+# - LIBRARY_PATH
```

Comment:
This implies some extra CLI switches im unaware of. When building, is this something I need to concern myself with? What does this mean?

Reply:
Nah, it just means that one can set those variables to specify alternate installation locations.